Source/WebCore/platform/mac/ScrollbarThemeMac.mm

```diff
@@ -53,11 +54,11 @@
 
 namespace WebCore {
 
-using ScrollbarToScrollerImpMap = HashMap<Scrollbar*, RetainPtr<NSScrollerImp>>;
+using ScrollbarSet = HashSet<Scrollbar*>;
```

Comment:
Our memory safe coding rules would say that this should be a hash of weak or checked pointers.

Reply:
Changed to checked pointer.